lib/rspec/mocks/error_generator.rb

```diff
@@ -304,6 +304,22 @@ def count_message(count, expectation_count_type=nil)
       def times(count)
         "#{count} time#{count == 1 ? '' : 's'}"
       end
+
+      def formatted_single_message(args)
+        "(" + args.first.map(&:inspect).join(", ") + ")"
+      end
```

Comment:
I somehow missed `format_args` completely.  `format_received_args` had the same code (basically) so I was trying to keep that working when refactoring. Using `format_args` instead should work much better.